install/empty_data.php

```diff
@@ -763,7 +763,17 @@ public function getEmptyData(): array
                 'mode' => CronTask::MODE_EXTERNAL,
                 'lastrun' => null,
                 'logs_lifetime' => 30,
-            ],
+            ], [
+                'id' => 43,
```

Comment:
Why using next id ?
```suggestion
                'id' => 42,
```

Reply:
The next ID is 43. The array is out or order.